.github/workflows/cut-release.yml

```diff
@@ -0,0 +1,59 @@
+name: "cut-release"
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'version'
+        required: false
+jobs:
+  cut_release:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout code 
+        uses: actions/checkout@v3
+      - name: "Build & test"
+        run: |
+          sudo apt-get install build-essential -y
+          ./autogen.sh
+          ./configure
+          make distcheck
```

Comment:
Can you also install the `fakeroot` package with apt-get and then use `fakeroot make distcheck`? This will produce tarballs with ownership `root/root` (0/0) instead of the current `runner/docker` (1001/122) which might produce a cleaner tree when unpacked as root at a target machine.